validator/db/kv/attester_protection.go

```diff
@@ -58,8 +59,16 @@ func (store *Store) CheckSlashableAttestation(
 		return sourceEpochsBucket.ForEach(func(sourceEpochBytes []byte, targetEpochBytes []byte) error {
 			existingSourceEpoch := bytesutil.BytesToUint64BigEndian(sourceEpochBytes)
 			existingTargetEpoch := bytesutil.BytesToUint64BigEndian(targetEpochBytes)
-			surrounding := att.Data.Source.Epoch < existingSourceEpoch && att.Data.Target.Epoch > existingTargetEpoch
-			surrounded := att.Data.Source.Epoch > existingSourceEpoch && att.Data.Target.Epoch < existingTargetEpoch
+			existingAtt := &ethpb.IndexedAttestation{
```

Comment:
Needed to adapt the input data to satisfy the input type of slashutil.IsSurround